File: problems/a/apaxiaaans/apaxiaaans.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

from asyncore import compact_traceback

def main():
    name = input()
    compact_name = ""
    for i in range(len(name)):
        if i == 0:
            compact_name += name[i]
        else:
            if name[i - 1] != name[i]:
                compact_name += name[i]
    print(compact_name)

    
if __name__ == '__main__':
    main()

File: problems/a/apaxiaaans/apaxiaaans.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

#define NAME_SIZE 251

int main(void) {
    char name[NAME_SIZE];

    if (scanf("%s", name) != 1) {
        printf("Failed to read name!");
    }

    for (int i=0; name[i] != '\0'; i++) {
        if (i == 0) {
            printf("%c", name[i]);
        }
        else {
            if (name[i - 1] != name[i]) {
                printf("%c", name[i]);
            }
        }
    }

    printf("\n");

    return 0;
}